js/game.js

```diff
@@ -0,0 +1,70 @@
+/** @type {HTMLCanvasElement} */
+
+class Game{
+    constructor(ctx, width, height, player){
+        this.ctx = ctx
+        this.width = width
+        this.height = height
+        this.player = player
+        this.intervalId = null
+        this.frames = 0
+        this.enemies = []
+    }
+
+    start(){
+        this.intervalId = setInterval(this.upDate, 1000/60) 
+    }
+
+     upDate = () => {
+        this.frames++
+        this.clear()
+        this.player.newPosition()
+        this.player.draw()
+        this.updateEnemies()
+        this.checkGameOver()
+    }
+
+    stop(){
+        clearInterval(this.intervalId)
+    }
+
+    clear(){
+        this.ctx.clearRect(0, 0, this.width, this.height)
+    }
+
```

Comment:
To make the score, that are a lot of ways of doing it, one of them is through the frames, so the score will increase depending on how long you are playing.

To do this you need to create a new method inside the game class:
```
  score(){
    const points = Math.floor(this.frames / 30);
    this.ctx.font = '22px monospace';
    this.ctx.fillStyle = 'black';
    this.ctx.fillText(`Score: ${points}`, 300, 50);
  }
```

And then you need to call this new method inside the update, otherwise the score will always be 0